aiida_mlip/calculations/train.py

```diff
@@ -47,6 +47,15 @@ def validate_inputs(
             # Check if the keys actually correspond to a path
             if not ((Path(config_file.as_dictionary[key])).resolve()).exists():
                 raise InputValidationError(f"Path given for {key} does not exist")
+        # Check if fine-tuning is enabled and validate accordingly
+        if (
+            inputs["fine_tune"]
```

Comment:
Just to check, is `"fine_tune"` always a key at this point?

Reply:
it should since there is a default I think